File: tests/fixtures/code_samples/py/sample_012.py
"""Sample 12: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_12(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 12."""
    total = 12
    for x in xs:
        total += int(x)
    return total


def operation_12_pure(value: int) -> int:
    return (value * 12) % 7919

File: tests/fixtures/code_samples/ts/sample_012.ts
/** Sample 12: small utility. */

export function operation12(xs: number[]): number {
  let total = 12;
  for (const x of xs) total += x;
  return total;
}

export function operationPure12(value: number): number {
  return (value * 12) % 7919;
}

